File: crates/cortex-bench/fixtures/langs/Retry.java
package langs;

public final class Retry {
    public static final int JAVA_RETRY_CAP = 8;

    private Retry() {}

    public static int scheduleJavaRetry(int attempt) {
        if (attempt >= JAVA_RETRY_CAP) {
            return 0;
        }
        return 1 << attempt;
    }
}

File: crates/cortex-bench/fixtures/langs/Retry.cs
namespace Langs;

public static class Retry
{
    public const int CsRetryCap = 8;

    public static int ScheduleCsRetry(int attempt)
    {
        if (attempt >= CsRetryCap)
        {
            return 0;
        }
        return 1 << attempt;
    }
}
